test/models/invoice.model.test.js

```diff
@@ -0,0 +1,51 @@
+'use strict'
+
+// Test framework dependencies
+const Lab = require('@hapi/lab')
+const Code = require('@hapi/code')
+
+const { describe, it } = exports.lab = Lab.script()
+const { expect } = Code
+
+// Thing under test
+const { InvoiceModel } = require('../../app/models')
+
+describe('Bill Run Model', () => {
+  describe('the $summarised() method', () => {
+    it("returns 'true' when the status is 'summarised'", async () => {
+      const instance = InvoiceModel.fromJson({ status: 'summarised' })
```

Comment:
Since `@summarised` isn't actually needed I've deleted the tests, and as they were the only tests on `InvoiceModel` I've deleted the test file entirely :grin: